google-cloud-spanner/src/main/java/com/google/cloud/spanner/SessionPool.java

```diff
@@ -2606,6 +2635,10 @@ enum Position {
 
   private AtomicLong numWaiterTimeouts = new AtomicLong();
 
+  @GuardedBy("lock")
+  private final AtomicReference<SettableApiFuture<MultiplexedSessionFuture>>
+      multiplexedSessionFuture = new AtomicReference<>(SettableApiFuture.create());
```

Comment:
nit: change the variable name to `currentMultiplexedSessionReference`. The current name seems to indicate that the field is of type `Future`.

Reply:
Done